CODEOWNERS

```diff
@@ -202,6 +202,7 @@ Kconfig*                                  @tejlmand
 /samples/sdfw/                            @hakonfam @jonathannilsen
 /samples/sdfw/ssf_client/                 @anhmolt
 /samples/suit/                            @tomchy @ahasztag
+/samples/suit/flash_companion/            @e-rk
```

Comment:
It will be safer to have at least two owners/directory, so you can add myself to the list :slightly_smiling_face:

Reply:
done